src/model/guild/audit_log.rs

```diff
@@ -285,6 +293,26 @@ mod u64_handler {
     }
 }
 
+mod option_u64_handler {
+    use super::*;
+
+    pub fn deserialize<'de, D: Deserializer<'de>>(des: D) -> StdResult<Option<u64>, D::Error> {
+        let s: Option<String> = Option::deserialize(des)?;
+        if let Some(s) = s {
+            return Ok(s.parse().map(Some).map_err(|e| de::Error::custom(e))?);
+        }
+
+        Ok(None)
+    }
+
+    pub fn serialize<S: Serializer>(num: &Option<u64>, s: S) -> StdResult<S::Ok, S::Error> {
+        match num {
+            Some(n) => s.serialize_u64(*n),
```

Comment:
Changing this to `Option::serialize(num, s)` compiles, and I'm pretty sure it does the same thing. Should I change it?